app/models/case_log.rb

```diff
@@ -442,6 +442,29 @@ def is_dpo
     created_by&.is_dpo
   end
 
+  delegate :service_name, :sensitive, :registered_under_care_act, :primary_client_group, :has_other_client_group, :secondary_client_group, :support_type, :intended_stay, :created_at, prefix: "scheme", to: :scheme, allow_nil: true
+  delegate :scheme_type, to: :scheme, allow_nil: true
+
+  def scheme_code
+    scheme&.id ? "S#{scheme.id}" : nil
+  end
+
+  def scheme_owning_organisation_name
+    scheme&.owning_organisation&.name
+  end
+
+  def scheme_managing_organisation_name
+    scheme&.managing_organisation&.name
+  end
+
+  delegate :postcode, :name, :units, :type_of_unit, :mobility_type, :startdate, prefix: "location", to: :location, allow_nil: true
+  delegate :location_admin_district, to: :location, allow_nil: true
+
+  # This is not the location_code in the db, location.id is just called code in the UI
+  def location_code
```

Comment:
I guess these can't because we're using `code` rather than `id`? Although actually that makes me wonder whether we shouldn't rather be consistent and refer to `id` when we mean `id`?

Reply:
So this is weird because the location's postcode is being recognised as its id, rather than the column actually named id - if I delegate the id from location then it actually returns the postcode for some reason.